make_level.py

```diff
@@ -0,0 +1,117 @@
+import sys
+import os
+
+cwd = os.getcwd()
+
+def write_init(skill_name, skill_path, level_name):
+    # If skills/skill/__init__.py doesn't exist, create a basic version (no levels)
+    if not os.path.exists(os.path.join(skill_path,"__init__.py")):
+        with open(os.path.join(skill_path,"__init__.py"), 'w+') as fp:
+            level_setup = "\n".join([
+                "import pkg_resources\n",
+                "from gitgud.skills.util import BasicLevel",
+                "from gitgud.skills.util import Skill\n"
+                "skill = Skill(",
+                "    '{}',".format(skill_name),
+                "    [",
+                "    ]",
+                ")\n",
+                ])
```

Comment:
For all of these, instead of newlines, can you do a new entry with a blank line, so
```
                ")\n",
```
Turns into
```
                ")",
                "",
```
I think it makes it clear, and that way we can see the visual separation here too

Reply:
Do you want this for the one line statements too?

Reply:
Yeah, we should be able to see what the files look like instead of just having the most compressed form